draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1028,12 +1028,14 @@ a cryptographic hash function that outputs b bits.
 ## Security considerations {#hash2base-sec}
 
 For security, hash2base should be collision resistant, and should map its
-input to a uniformly random element of F.
-
-To ensure sufficient collision resistance, the number of bits output by
-the hash function H should be b >= 2 * k, where k is the target security
-level in bits. For example, for 128-bit security, b >= 256 bits; in this
-case, SHA256 would be an appropriate choice for H.
+input to a uniformly random element of F. To this end, hash2base requires 
+a cryptographic hash function H which satisfies the following properties:
+
+1. The number of bits output by H should be b >= 2 * k for sufficient collision 
+resistance, where k is the target security level in bits. (This is needed for a 
+birthday bound of approximately 2^(-k).) For example, for 128-bit security, b >= 256 
+bits; in this case, SHA256 would be an appropriate choice for H.
+2. H must output a uniformly random bit string. That is, it must be a random oracle.
 
 Ensuring that the hash2base output is a uniform random element of F requires
 care, even when H outputs a uniformly random string, as is generally assumed
```

Comment:
"as is generally assumed (in other words..."

is redundant with the new point (2). Probably can just cut from ", as is" to the end of the sentence.

Reply:
Yep, dropped!